.github/workflows/ai-scanner.yml

```diff
@@ -0,0 +1,23 @@
+name: AI Code Review
+
+on:
+    push:
+      branches: [ "main" ]
+    pull_request:
+      branches: [ "main" ]
+    schedule:
+      - cron: "29 4 * * 3"
```

Comment:
As it stands: this job will run at 4:29 AM on every Wednesday.

Reply:
ya, i wwas thinking of leaving in a cron job, but removed it -- it's probably not necessary.  at least for now